### ISSUE TITLE
Exception console spam

### ISSUE THREAD

**Issue by LordSheep** [montlikadani/TabList#36]
Used the plugin AdvancedBan and was testing it out. Everything is fine with the console until I use the command /ban on someone and then it spams some text (relating to tablist) every second in the console.

The tablist works fine but my console is just getting too spammed.

**Console**
https://pastebin.com/WbeWq2Az

**Config**
https://pastebin.com/FKcQcmkm 

For placeholderapi I have 3 expansions installed: Vault, ASkyBlock, Essentials

**Comment by montlikadani** [comment]
Do you use the latest version of PlaceholderAPI?

**Comment by LordSheep** [comment]
Yes, I use version 2.9.1.

**Comment by LordSheep** [comment]
Still spams console right after a ban

https://pastebin.com/C9SKLLtU

**Comment by montlikadani** [comment]
This will be a PAPI problem because there is no bug in my code that could cause what the console writes have already verified and leads to PAPI.

Send this bug to PAPI here and describe that **when a player banned from the server throw a long bug in the console.**
send to: https://github.com/PlaceholderAPI/PlaceholderAPI/issues
If ask the owner, tell him I'm using TabList, but that does not cause the error because the TabList owner has checked it, but this error has led to PAPI.

**Comment by LordSheep** [comment]
Okay, thanks.

**Comment by darbyjack** [comment]
Can't really see anything on our end. It looks like it's failing to get the UUID of a player from your plugin, at least, that's the line that is erroring there in the NPE. The latest version of PAPI does switch to OfflinePlayer so you might want to try switching to that too in order to handle the new Placeholder system.

**Comment by LordSheep** [comment]
I think I found the problem. It's this line in the config: &fIsland Border: %isborder_status%&r

It was the placeholder for this plugin: https://www.spigotmc.org/resources/askyblock-acidisland-island-border.56320/ 

The placeholder worked fine and showed the status of the island border.

I tried removing it and as soon as I banned someone there was no spam in console. I put it back, restarted the server and banned a player and it did it again. So that line was the problem. I think i'll just remove it as it isn't really necessary.
